### ISSUE TITLE
isAbout behaviour

### ISSUE THREAD

**Issue by djarecka** [sanuann/nidmterms-ui#42]
@satra @nqueder - I'm trying to extend the search, but had some issue with `isAbout` behavior and realized that it's the same in `https://incf-nidash.github.io/nidmterms-ui/?#/annotate`. Can you please confirm that this is an issue, and it should look differently.
For example, when I use the term `language` and choose one concept from the right, it is not checked in `isAbout` list, is that ok?
<img width="1394" alt="Screen Shot 2022-05-29 at 5 45 36 PM" src="https://user-images.githubusercontent.com/1662746/170892512-7e7ac8ae-3cdd-486d-9c37-f1896b70d1ff.png">

Also, when I click `Broaden the search` the list on the right is longer but the list in `isAbout` stays the same...
And when I click one of the term from the "extended list" all of them disappear. 

Is it just me who has these problems??

**Comment by satra** [comment]
nope, those are similar behaviors for me. i think those are elements that need to be fixed.
